src/Consts/CLASSES.js

```diff
@@ -0,0 +1,325 @@
+const CLASSES = [
```

Comment:
delete this file. We dont need it any more
we use now: src/Consts/classesConst.js